internal/lookoutingester/instructions/instructions.go

```diff
@@ -282,6 +284,22 @@ func handleJobSucceeded(ts time.Time, event *armadaevents.JobSucceeded, update *
 	return nil
 }
 
+func handleJobRunPreempted(ts time.Time, event *armadaevents.JobRunPreempted, update *model.InstructionSet) error {
+
+	jobUpdate := model.UpdateJobInstruction{
```

Comment:
I don't think you need to fail the job here, you get a JobFailed message for that.

Reply:
will remove